### ISSUE TITLE
Weird issue with no styling on DataTable

### ISSUE THREAD

**Issue by XaeroDegreaz** [TheCoder4eu/BootsFaces-OSP#247]
All of the Bootstrap styling works everywhere else within my application, but no matter where I place the DataTable using Bootsfaces 0.8.0, it just has no styling o_O

I was shocked to see the awesome improvements to the DataTable, so I looked on the Bootsfaces site to see what was new. I noticed there is Bootsrap styling on all of the components of the table in your example but when I use the table, it looks unstyled:

```xml
<b:panel>
	<f:facet name="heading">
		<b><h:outputText />Agents to be Reviewed</b>
	</f:facet>
	<b:dataTable
		styleClass="table table-striped table-bordered"
		value="#{indexPage.agentsToBeReviewed}"
		var="annotations">
		<h:column>
			<f:facet name="header">Team</f:facet>
			#{annotations.team.name}
		</h:column>
	</b:dataTable>
</b:panel>
```
![image](https://cloud.githubusercontent.com/assets/312459/12331874/2b548ed0-bab1-11e5-82f9-fa182260bfcb.png)

Is there some other dependency I need to add?

**Comment by stephanrauh** [comment]
I'm glad you like our datatable! It's an unfinished prototype, so bear with us. I know it's far from being perfect.

As for your question: at first glance, your screenshot looks OK to me. The only thing that's odd is the paginator at the bottom. What do you refer to when you say there's no styling?

**Comment by XaeroDegreaz** [comment]
The drop down menu, and the filter input are not styled with bootstrap.
Neither are the previous, 1, 2, .... 5, next buttons.

If you compare my screenshot to the datatable shown on the Bootsfaces
website demo, you can clearly see that mine is missing all of the Bootstrap
styling for some reason.

I understand that the new datatable is unfinished, but I wanted to make
sure that it wasn't something I was overlooking.

Everything else on Bootsfaces works wonderfully, and I can tough it out;
the functionality of the page drop down and the filter is freaking sweet so
I'll gladly wait for the fix on the actual styling.

**Comment by stephanrauh** [comment]
Now that you say I... I looked at it all the time without recognizing there was something missing which already worked before. I simply forgot to add the JavaScript code. I've uploaded a new developer snapshot of BootsFaces to Maven Central. See #151 on how to get it.

As a quick workaround, simply add this code snippet to your XHTML file:

```xml
<h:outputScript library="bsf" name="js/jquery.dataTables.min.js" />
<h:outputScript library="bsf" name="js/dataTables.bootstrap.min.js" />
```

I added it to the body - maybe it works in the header to, but I didn't try.
